src/java.base/share/classes/java/lang/classfile/instruction/CharacterRange.java

```diff
@@ -70,17 +74,17 @@ public sealed interface CharacterRange extends PseudoInstruction
     /**
      * A flags word, indicating the kind of range.  Multiple flag bits
      * may be set.  Valid flags include
-     * {@link jdk.internal.classfile.Classfile#CRT_STATEMENT},
-     * {@link jdk.internal.classfile.Classfile#CRT_BLOCK},
-     * {@link jdk.internal.classfile.Classfile#CRT_ASSIGNMENT},
-     * {@link jdk.internal.classfile.Classfile#CRT_FLOW_CONTROLLER},
-     * {@link jdk.internal.classfile.Classfile#CRT_FLOW_TARGET},
-     * {@link jdk.internal.classfile.Classfile#CRT_INVOKE},
-     * {@link jdk.internal.classfile.Classfile#CRT_CREATE},
-     * {@link jdk.internal.classfile.Classfile#CRT_BRANCH_TRUE},
-     * {@link jdk.internal.classfile.Classfile#CRT_BRANCH_FALSE}.
+     * {@link java.lang.classfile.ClassFile#CRT_STATEMENT},
```

Comment:
Maybe it would be better to organize the links in a bullet list?

Reply:
Yes, fixed, thanks.